### PR TITLE
Remove '-java' postfix from references

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: docs/src/main/paradox/java/http/routing-dsl/directives/security-directives/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 @@@
 
-<a id="authentication-vs-authorization-java"></a>
+<a id="authentication-vs-authorization"></a>
```
Comment (jrudolph) [inline docs/src/main/paradox/java/http/routing-dsl/directives/security-directives/index.md]: Do we still need those at all?

Comment (raboof) [reply docs/src/main/paradox/java/http/routing-dsl/directives/security-directives/index.md]: only when they don't correspond 1-to-1 with the heading I believe... would be nice to go through those but then I'd like to have https://github.com/lightbend/paradox/issues/120 first

Comment (jrudolph) [reply docs/src/main/paradox/java/http/routing-dsl/directives/security-directives/index.md]: I see, makes sense.

**Comment by ktoso** [comment]
LGTM

**Comment by ktoso** [comment]
#1624

**Pull request merged**